fighthealthinsurance/chat_interface.py

```
@@ -326,11 +337,214 @@ async def _call_llm_with_actions(
+                await self.send_status_message(cleaned_response)
+
+                try:
+                    from fighthealthinsurance.medicaid_api import is_eligible
+
+                    await self.send_status_message(
+                        "Processing medicaid elgibility data"
+                    )
+                    (eligible_2025, eligible_2026, medicare, alternatives, missing) = (
+                        is_eligible(**loaded)
+                    )
+                    info_text = f"We're helping figure out if someone is likely eligible for medicaid. Be clear this is an approximation and they'll need to confirm with the state to be sure."
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)